### ISSUE TITLE
Dead keys not registered

### ISSUE THREAD

**Issue by kurt22kurt** [8LWXpg/RegExHotstring#6]
---------
Description
-----------

I've encountered an issue while using the RegExHotstring.ahk script, where dead keys are not being registered as expected. Dead keys, such as '^', '¨', and '´', are not modifying characters when used in conjunction with the script.

Thank you for your assistance.

-----------
Steps to reproduce
-----------
1. Download RegExHotstrings.ahk and set up following script:
line 1: `#Include RegExHotstrings.ahk `
2. Use a French keyboard
  Windows 10:  
  2.a. Settings >Time & Language > Language > Preferred languages > [Language] > Options > Add a keyboard > French
 2.b. To switch between keyboard layouts, press Alt+Shift.
3. Attempt to use dead keys (e.g., '^', '¨', '´') with the script active.
4. Observe that the dead keys do not modify characters as intended.

--------------
Dead keys
--------------
Dead keys are keys on a keyboard that don't produce a character on their own when pressed, but instead modify the character that is entered when you press another key immediately after the dead key. These keys are commonly used in languages that require diacritics or accent marks to be placed on top of certain characters. Keyboards designed for languages like French, Spanish, German, or Portuguese often include dead keys to make it easier to input accented characters. 

For example, the '^' key is a dead key for adding circumflex accents (ex. â) , the '¨' key is a dead key for adding diaeresis (umlaut) accents (ex. ä). When you press one of these dead keys and then press a letter, the dead key modifies the letter accordingly.

**Comment by 8LWXpg** [comment]
I cannot replicate the issue. What keyboard layout are you using?

I have tested the international keyboard and the French keyboard (which the GIF shows).

![output](https://github.com/8LWXpg/RegExHotstring/assets/105704427/da839423-ec84-4e93-a5b3-b30a8907dfe1)

The tooltip shows current input received from InputHook.

However, I do notice that when you press dead keys twice (for example, “`”), it will send two keys at once, which is suppressed and will not send.

**Comment by kurt22kurt** [comment]
Firstly, I sincerely apologize for any confusion caused by my previous comment. It appears I provided inaccurate information in my initial description of the problem, and I appreciate your patience in seeking clarification.

Upon revisiting the issue and carefully considering your response, I realize that my explanation was not accurate. The root cause of the problem was indeed the behavior you mentioned: two keys send at once are suppressed and not actually transmitted.

My actual issue was not being able to input sequences like "^v" while your script was active. I realize now that I misinterpreted the behavior and incorrectly described it in my previous message.
